pkg/v3/observer.go

```diff
@@ -28,12 +28,12 @@ type Runner interface {
 	CheckUpkeeps(context.Context, ...ocr2keepers.UpkeepPayload) ([]ocr2keepers.CheckResult, error)
 }
 
-type Observer[T any, R any] struct {
+type Observer[T any] struct {
```

Comment:
if you want you can also cleanup the input type which is always an UpkeepPayload